### PR TITLE
Create testfile2

### PR DESCRIPTION
**Pull request by torandu**
This change adds some text

### CODE REVIEW

File: testfile2 [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# lorem ipsum
+
+Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed nec auctor neque, quis fermentum purus. Proin tincidunt lectus lorem, ac venenatis purus rutrum sed. Duis bibendum ipsum justo, sit amet venenatis lorem auctor a. In et lorem malesuada, placerat dolor eu, ullamcorper nunc. Morbi vel volutpat nibh. Sed malesuada vitae sapien eget fermentum. Mauris risus risus, mollis id mollis sed, condimentum eu libero. Orci varius natoque penatibus et magnis dis parturient montes, nascetur ridiculus mus. Proin eu maximus urna. Nunc ultricies lectus vitae risus eleifend, nec facilisis ipsum condimentum.
```
Comment (m1napatel) [inline testfile2]: This line needs a change

Comment (torandu) [reply testfile2]: no, this is how it should work

Comment (m1napatel) [reply testfile2]: You, sure? Surely it needs blah init!

**Comment by torandu** [comment]
@m1napatel check this out

**Pull request merged**